.github/workflows/cargo_clippy.yml

```diff
@@ -25,10 +25,10 @@ jobs:
       - uses: actions-rs/clippy-check@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-targets --features std
```

Comment:
Just to make sure: does this mean that the clippy workflow won't test examples but only the main code?

Reply:
Examples will still be checked with `cargo check`. Since clippy is running with `--features std`, examples don't compile. So yes! To have examples compile with std, we'd need to make them more verbose with lots of compile flag switching.